sdk/sui-owned-object-pools/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "suioop",
+  "version": "2.3.2",
+  "description": "Sui Owned Object Pools is a library that provides a set of tools for managing multiple concurrent transactions on the Sui network to help avoiding object equivocation and locking",
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "type": "commonjs",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    }
+  },
+  "files": [
+    "dist",
+    "src",
+    "CHANGELOG.md"
+  ],
+  "scripts": {
+    "format": "prettier --write '**/*.{ts,tsx,js,jsx,json,css,md}'",
+    "lint": "eslint --ext .ts --fix --max-warnings 0 src",
+    "test": "jest test/unit/executorServiceHandler.test.ts && jest test/unit/pool-tx-execution.test.ts && jest test/unit/pool-init.test.ts",
```

Comment:
We probably want to use some sort of pattern matching here rather than running each test file explicitly here.

If possible I'd also replace jest with vitest which we use in most of our other tests